cmd/release-controller/sync_verify_prow.go

```diff
@@ -62,6 +62,9 @@ func (c *Controller) ensureProwJobForReleaseTag(release *Release, verifyName str
 		return nil, terminalError{err}
 	}
 
+	if isAggregatedJob {
+		periodicConfig.Name = fmt.Sprintf("%s-%s", jobName, verifyName)
```

Comment:
can this be moved to the lower isAggregatedJob block?

Reply:
Unfortunately no.  It *must* happen after fetching and validating the `periodicConfig`.

Reply:
can't be done near line 74>

Reply:
At line 68, the `periodicConfig` is converted over to `ProwJobSpec`, that sets the name from then on.